### PR TITLE
Added blog post for week 10

### PR DESCRIPTION
**Pull request by m-agour**
My blog post for week 10.

### CODE REVIEW

**Review by xtanion — COMMENTED** [review]
Hi @m-agour, Really nice blog. No typos, PTAL at my suggestion below. Feel free to change them accordingly.
Thanks

File: docs/source/posts/2022/2022-08-23-week-10-mohamed.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+What is coming up next week?
+----------------------------
+
+- Finish the hierarchical order animation support `PR`_.
+
+- Explain tutorials in more detail. See this `issue`_.
+
+- Fix issues discovered by Serge in this `review`_.
+
+- Fix ``lightColor0`` being `hard-set`_ to ``(1, 1, 1)``, instead find a way to get this property the same way other polygon-based actor gets it.
```
[the remark this answers is not in the document]
Comment (m-agour) [reply docs/source/posts/2022/2022-08-23-week-10-mohamed.rst:35]: LOL

**Review by ganimtron-10 — COMMENTED** [review]
Hello @m-agour , 
Great Blog, LGTM!

File: docs/source/posts/2022/2022-08-23-week-10-mohamed.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
+Week 9: Animating primitives of the same actor
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-08-23-week-10-mohamed.rst:1]: week 10

**Review by skoudoro — COMMENTED** [review]
apart from the typo below. LGTM. 

I will merge after your commit

**Comment by m-agour** [comment]
Thank you @skoudoro @ganimtron-10 @xtanion for you review

**Comment by skoudoro** [comment]
Thank you for the update

**Pull request merged**